### PR TITLE
Fixed notebook as reported in #970

### PR DESCRIPTION
**Pull request by cmastalli**

### CODE REVIEW

**Comment by hrp2-14** [comment]
Hi ! This project doesn't usually accept pull requests on master. If this wasn't intentionnal, you can change the base branch of this pull request to devel (No need to close it for that). Best, a bot.

File: examples/notebooks/introduction_to_crocoddyl.ipynb [hunk 1]
```diff
@@ -303,17 +304,19 @@
...
    "outputs": [],
    "source": [
     "# Create the cost functions\n",
     "target = np.array([0.4, 0., .4])\n",
-    "Mref = crocoddyl.FrameTranslation(robot_model.getFrameId(\"gripper_left_joint\"), target)\n",
     "state = crocoddyl.StateMultibody(robot_model)\n",
-    "goalTrackingCost = crocoddyl.CostModelFrameTranslation(state, Mref)\n",
-    "xRegCost = crocoddyl.CostModelState(state)\n",
-    "uRegCost = crocoddyl.CostModelControl(state)\n",
+    "frameTranslationResidual = crocoddyl.ResidualModelFrameTranslation(state,\n",
```
Comment (proyan) [inline examples/notebooks/introduction_to_crocoddyl.ipynb:314]: Residual Models are not yet released on master branch. This won't work.

**Review by proyan — CHANGES_REQUESTED** [review]
Thanks for the fix Carlos.

File: examples/notebooks/introduction_to_crocoddyl.ipynb (hunk 1 above)
Comment (proyan) [reply examples/notebooks/introduction_to_crocoddyl.ipynb:314]: Ah the branch was changed to devel. My bad

**Review by proyan — APPROVED** [review]

**Pull request merged**